snap-core.cabal

```diff
@@ -195,7 +195,7 @@ Library
                    network     >= 2.3 && < 2.6
 
   if impl(ghc >= 7.6)
-    build-depends: unix-compat >= 0.3 && < 0.6
+    build-depends: unix-compat >= 0.3 && < 0.7
```

Comment:
This line looks redundant as it does not further constrain `unix-compat`.

Reply:
Indeed. Updated :)